app/com/linkedin/drelephant/tuning/AbstractFitnessManager.java

```diff
@@ -162,15 +161,12 @@ protected void resetParamSetToCreated(JobSuggestedParamSet jobSuggestedParamSet,
       logger.debug("Resetting parameter set to created: " + jobSuggestedParamSet.id);
       jobSuggestedParamSet.paramSetState = JobSuggestedParamSet.ParamSetStatus.CREATED;
       jobSuggestedParamSet.save();
-      JobExecution latestJobExecution = JobExecution.find.where()
```

Comment:
This was done earlier to mitigate problem in update for modifying an object which was updated. Are we not encountering that problem now ?

Reply:
There were multiple places ,where jobexecution was updated earlier . I changed it to update it once , so error should occur now